src/Core/Animation/HandleSystem.hpp

```diff
@@ -15,30 +15,30 @@ namespace Animation {
 using Label = std::string;
 
 /**
- * The Handle class describes the base handle structure used for deformation.
+ * The HandleSystem class describes the base handle structure used for deformation.
  */
-class RA_CORE_API Handle
+class RA_CORE_API HandleSystem
```

Comment:
If I understand well, HandleSystem is an array one name and labelled transform, since pose is an array of transform, and the only other data in HandleSystem are m_label and m_name
So the name sounds weird. I do not get the "system" part of it.
Why not NamedPose ?
Actually Pose might name TransformArray, and HandleSystem will be NamedTransformArray

So a Skeleton is NamedTransformArray which is a TranformArray.
Well, not sure neither ...

Reply:
+1
This class is not a system. So, why call it so ?